spec/requests/users_spec.rb

```diff
@@ -20,17 +20,17 @@
 
   describe 'GET /users/:id' do
     it 'return a successful response' do
-      get '/users/10'
+      get '/users/95'
       expect(response).to have_http_status(200)
```

Comment:
- [ ] Well done, you are very close @nilab-khwaja , it seems like you did work according to the previous code reviewer's suggestion but your spec tests are still failing, while making some changes, I think something happened, please make sure that your test cases should pass, because `2` tests are again failing for the `show` and endpoint for the `user` controller.

![image](https://github.com/nilab-khwaja/blog-app/assets/66476329/11e8e4b9-3113-47b7-b8b2-5a7b41d596a1)

Reply:
@skyv26  Hi , Thanks for your review, this is becuase there is no such id in your database. but there is no error on my side 
<img width="578" alt="Capture" src="https://github.com/nilab-khwaja/blog-app/assets/122613936/361c26a4-cbb1-4b09-a632-0122e903cdfb">